sphinx/source/docs/user_guide/tools.rst

```diff
@@ -561,6 +562,218 @@ shown below:
+on one or more renderers by editing the underlying
+``ColumnDataSource`` data. Like other drawing tools, the renderers
+that are to be edited must be supplied explicitly as a list. When
+drawing a new box the data will always be added to the
+``ColumnDataSource`` on the first supplied renderer.
+
+The tool will automatically modify the columns on the data source
+corresponding to the ``x``, ``y``, ``width`` and ``height`` values of
+the glyph. Any additional columns in the data source will be padded
+with the declared ``empty_value``, when adding a new box.
+
+.. raw:: html
+
+    <img src="http://bokeh.pydata.org/static/box_draw_keyboard_optimized.gif"
+     width='400px' alt="Animation showing box draw, select and delete actions">
+
+The animation above shows the supported tool actions, highlighting
+mouse actions with a circle around the cursor and key strokes by
+showing the pressed keys. The supported actions include:
+
+* Add box: Click and hold the mouse button down, dragging the mouse
+  from the starting point to the end point then release the
+  button. This will define the two opposite corners of the rectangle.
+
+* Select box: Tap on an existing box. To select multiple hold SHIFT
+  key while tapping.
+
+* Delete box: Select one or more boxes then press DELETE key.
+
+.. bokeh-plot:: docs/user_guide/examples/tools_box_draw.py
+    :source-position: none
+
+
+PointDrawTool
+~~~~~~~~~~~~~
+
+* name: ``'point_draw'``
+* menu icon: |point_draw_icon|
+
+The PointDrawTool allows adding, dragging and deleting point-like
+glyphs (of ``XYGlyph`` type) on one or more renderers by editing the
+underlying ``ColumnDataSource`` data. Like other drawing tools, the
+renderers that are to be edited must be supplied explicitly as a
+list. Any newly added points will be inserted on the
+``ColumnDataSource`` of the first supplied renderer.
+
+The tool will automatically modify the columns on the data source
+corresponding to the ``x`` and ``y`` values of the glyph. Any
+additional columns in the data source will be padded with the declared
+``empty_value``, when adding a new point.
+
+.. raw:: html
+
+    <img src="http://bokeh.pydata.org/static/point_draw_keyboard_optimized.gif"
+     width='400px' alt="Animation showing point draw, drag, select and delete actions">
+
+The animation above shows the supported tool actions, highlighting
+mouse actions with a circle around the cursor and key strokes by
+showing the pressed keys. The supported actions include:
+
+* Add point: Tap anywhere on the plot.
+
+* Select point: Tap on an existing point. To select multiple hold
+  SHIFT key while tapping.
+
+* Delete point: Select one or more points then press DELETE key.
+
+* Move point: Click and hold the mouse down point, drag it, then
+  release the mouse to complete the action. To move multiple points
+  select them and hold SHIFT key while dragging one of them.
+
+.. bokeh-plot:: docs/user_guide/examples/tools_point_draw.py
+    :source-position: none
```

Comment:
In this and the other examples, the tool is not active by default. I tried to just click to add points right away and nothing happened. Immediate thought was "what's broken". Suggest making the relevant edit tools active by default in these examples.

Reply:
Also in this specific example I think the table should go above, or the plot has to be made much narrower

Reply:
<img width="923" alt="screen shot 2018-01-25 at 12 08 20" src="https://user-images.githubusercontent.com/1078448/35404408-7d13995e-01c8-11e8-8199-de30737447ae.png">